odl/contrib/datasets/ct/mayo.py

```diff
@@ -47,43 +51,48 @@ def _read_projections(folder, indices):
 
     file_names = file_names[indices]
 
-    data_array = None
-
     for i, file_name in enumerate(tqdm.tqdm(file_names,
                                             'Loading projection data')):
         # read the file
-        dataset = dicom.read_file(folder + '/' + file_name)
+        try:
+            dataset = pydicom.read_file(folder + os.path.sep + file_name)
```

Comment:
The original version was wrong regarding path building. Please use the Python functionality intended for that. For now (still Py2 compatible 😒) that would be `path.join` with `from os import path`.